examples/biosnoop.rs

```diff
@@ -66,7 +66,7 @@ fn do_main(runnable: Arc<AtomicBool>) -> Result<(), BccError> {
         .attach(&mut bpf)?;
     Kprobe::new()
         .handler("trace_req_completion")
-        .function("blk_account_io_completion")
+        .function("blk_account_io_done")
```

Comment:
This should not be changed in this PR. Please revert changes to this file.

Reply:
From kernel 5.8+, `blk_account_io_completion` is no longer defined in `block/blk.h` and kprobe failed to catch it with kernel 5.8+. See the kernel source search result: https://elixir.bootlin.com/linux/v5.8/C/ident/blk_account_io_completion

BCC also replaced `blk_account_io_completion` by `blk_account_io_done`, see: ttps://github.com/iovisor/bcc/issues/2091

So I have to replace `blk_account_io_completion` by `blk_account_io_done`, otherwise it won't work with kernel 5.8+.

Reply:
This change is not related to adding RingBuf support. It belongs in a separate PR.